docs/modules/lodash-underscore.md

```diff
@@ -0,0 +1,9 @@
+# `lodash` / `underscore`
+
+`lodash` and `underscore` have not been updated for many years and almost all methods can be replaced with modern JavaScript built-in methods, the rest - with simple helpers.
+
+Extracted from `lodash` packages like `lodash.%methodname%` are not maintained, deprecated, haven't been updated for many more years, and a part of them have significant security issues and bugs. They are no longer safe for usage.
```

Comment:
I'm not sure that makes sense to explain why a package is not safe for usage if it's deprecated and / or not maintained 8+ years. About security issues and bugs... I was faced with a dozen different security issues and bugs in those packages, but I haven't a list. I have some examples, but I don't think that makes sense to add only those examples since users could think that's all and those packages are safe in other cases.

Reply:
Sure, for security cases, we could use a link to, for example, Snyk, like https://security.snyk.io/package/npm/lodash/4.17.9 - but it's the link to the main package since we can't collect all methods packages in one place and... for what version? They were abandoned at different times.

Reply:
i think i would reword it slightly still, since lodash isn't deprecated afaik, and does still receive commits (but rarely)

i'd just point out that it rarely receives updates and has been known to have security issues in _some versions_ (latest has none it seems)

i wouldn't say it is no longer safe for usage since technically it is. i understand our opinion is its a risky dependency but factually the latest version is probably safe to use

what do you think? i hope im making sense

Reply:
> lodash isn't deprecated afaik, and does still receive commits

Only the main package, not the rest. Something was abandoned 11 years ago, something 8, etc.

Reply:
ah my mistake! i totally missed the first part of the sentence

all good in that case 👍